### ISSUE TITLE
Vue.vscode-typescript-vue-plugin not working in a Cypress component test

### ISSUE THREAD

**Issue by segevfiner** [johnsoncodehk/volar#2137]
I can't seem to get `Vue.vscode-typescript-vue-plugin` to recognize a vue SFC import in a Cypress component test in this repo https://github.com/segevfiner/vue-component-test, see `cypress/component/HelloWorld.cy.ts`, it doesn't seem to recognize the `.vue` import in that file. I'm not sure if I configured something incorrectly for it to do so.

```json
[{
	"resource": "/Users/segevfiner/junk/vue-component-test/cypress/component/HelloWorld.cy.ts",
	"owner": "typescript",
	"code": "2307",
	"severity": 8,
	"message": "Cannot find module '../../src/components/HelloWorld.vue' or its corresponding type declarations.",
	"source": "ts",
	"startLineNumber": 1,
	"startColumn": 24,
	"endLineNumber": 1,
	"endColumn": 61
}]
```

**Comment by johnsoncodehk** [comment]
This is a bug of TS plugin and not easy to fix, as a workaround you can consider to use takeover mode for now.

**Comment by segevfiner** [comment]
Figured as much. But it does feel like something worth fixing at some point.

**Comment by johnsoncodehk** [comment]
Seems to have been fixed by https://github.com/johnsoncodehk/volar/commit/062895a2eba72ea659b98e23fcc5bbc443af7e7d (Released when 1.0.16)
